core/models/payload.go

```diff
@@ -61,11 +61,13 @@ type RequestDetails struct {
 	Scheme      string
 	Query       map[string][]string
 	Body        string
+	FormData    map[string][]string
 	Headers     map[string][]string
 	rawQuery    string
 }
 
 func NewRequestDetailsFromHttpRequest(req *http.Request) (RequestDetails, error) {
+	req.ParseForm()
```

Comment:
I found that this will break capture mode. Since it has read the body into a PostForm, 
L75: 
```
reqBody, err := util.GetRequestBody(req)
```
will return an empty body, and the captured simulation will be missing the request payload. 
You can try with this curl command: 
```
curl -X POST https://reqbin.com/echo/post/form --header 'Content-Type: application/x-www-form-urlencoded' --data 'param1=value1&param2=value2' --proxy localhost:8500
```

Reply:
We save the captured simulation using this function: https://github.com/SpectoLabs/hoverfly/blob/master/core/hoverfly_funcs.go#L291

That need to be updated to save the form data in the body field.  So that when you switch back to simulate mode, the same request will get matched.

Reply:
Fixed in the latest commit! 😄

Reply:
Ok manual testing it in webserver mode raises an error after the capture, will dive into that as well